clientside/config.json

```diff
@@ -0,0 +1,63 @@
+{
+    "compile":[
+        {
+            "files": [
+                "src/Constants.js",
+                "src/GameMaker.js",
+                "src/UIController.js",
+                "src/Main.js"
+            ],
+            "dist": {
+                "main": "WebViewer.js",
+                "minified": "WebViewer.min.js"
+            },
+            "inPreview": true
+        },
+        {
+            "files": [
+                "src/Constants.js",
+                "src/Entity.js",
+                "src/EntityColor.js",
+                "src/EntityHandler.js",
+                "src/WorkerDataLoader.js",
+                "src/WorkerMain.js"
+            ],
+            "dist": {
+                "main": "WebViewerWorker.js",
+                "minified": "WebViewerWorker.min.js"
+            },
+            "inPreview": false
+        }
+    ],
+    "doc":{
+        "jsdoc": "docs/jsdoc/",
+        "template": "scripts/compile/markdown.mustache"
+    },
+    "preview":{
+        "template": "scripts/compile/preview.mustache",
+        "comment": "scripts/compile/comment.mustache",
+        "title": "RCRS Web Viewer",
+        "modules": [
+            "jquery/dist/jquery.min.js",
+            "jq-ajax-progress/src/jq-ajax-progress.min.js",
+            "bootstrap/dist/js/bootstrap.min.js",
+            "earcut/dist/earcut.min.js",
+            "canvas-drawer/dist/CanvasDrawer.min.js"
+        ],
+        "path": "preview/",
+        "param":{
+            "devel" : {
+                "workerFile": "../src/WorkerLoader.js"
+            },
+            "dist" : {
+                "workerFile": "../dist/WebViewerWorker.js"
+            },
+            "dist.min" : {
+                "workerFile": "../dist/WebViewerWorker.min.js"
+            }
+        }
+    },
+    "dist_path": "dist/",
+    "version": "0.2",
+    "homepage": "https://github.com/roborescue/rcrs-web-viewer"
+}
```

Comment:
Please fix newline at end of files in all files.

Reply:
Thanks for the review,
I fixed this in two commits (3aac43216a6b64c63afc0fba9b72dcf06d9d2304 and 0ea2ad0b5f03a73fe452633dc6e94061de428aea) after reading your review.